### DISCUSSION TITLE
Repulse should close the mouse trail faster

### DISCUSSION THREAD

**Discussion by motion-work** [matteobruni/tsparticles#1481] [Q&A]
Is there a way to close the trail behind the mouse very fast when using the repulse mode (onHover). Like in the fish swarm example: [https://www.youtube.com/watch?v=jsnwQZ3h7XE](https://www.youtube.com/watch?v=jsnwQZ3h7XE)

**Comment by matteobruni** [comment]
Hi @motion-work,

Actually I don’t think is possible but it’s a great feature to achieve. I’ll have a look about that soon!

**Reply by motion-work** [reply]
sounds great! thanks for the quick response.
